test/accountBalance/AccountBalance.getPositionValue.test.ts

```diff
@@ -308,27 +307,28 @@ describe("AccountBalance.getTotalPositionValue", () => {
         expect(await accountBalance.getTotalPositionSize(alice.address, baseToken.address)).eq(
             parseEther("-1.465065799750044640"),
         )
-        // 152.711203 * -1.465065799750044640 = -223.731960754
+
+        // 152.711203175725718234 * -1.465065799750044640 = -223.7319610114
```

Comment:
Need some help: not sure where causes the precision to change

Reply:
Position value is calculated by mark price instead of by index twap. We print the mark price through console, and the value here is 152.711203175725718234. It's different from the index price above `152.711203`.

Reply:
Got it!

Reply:
what if we write it like above section, use `const markPrice = await accountBalance.getMarkPrice(baseToken.address)` to caluclate account value?